pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs

```diff
@@ -108,7 +108,11 @@ private void DisplayData()
             {
                 for (var i = 0; i < numColumns; i++)
                 {
-                    dataGrid.Columns[i].ToolTipText =
+                    // In the case when we don't have user provided headers, we still want localized headers that can be translated,
+                    // this replaces the autogenerated strings with a localized version
+                    dataGrid.Columns[i].HeaderText = string.Format(Resources.ImportTransitionListColumnSelectDlg_DisplayData_Column_) + (i+1).ToString();
+
```

Comment:
Probably cleaner and more general to do this:
string,Format("Column {0}", i+1)

That way if the local language has a convention like "3 column, this is" it still works.

Reply:
(So they'd swap in a string like "{0} column, this is")

Reply:
Should be fine now